server.go

```diff
@@ -114,14 +113,15 @@ func (server *Server) Wait() {
 
 // Stop stops the server
 func (server *Server) Stop() {
+	defer log.Info("Server stopped")
 	for serverEntryPointName, serverEntryPoint := range server.serverEntryPoints {
 		ctx, cancel := context.WithTimeout(context.Background(), time.Duration(server.globalConfiguration.GraceTimeOut)*time.Second)
-		go func() {
-			log.Debugf("Waiting %d seconds before killing connections on entrypoint %s...", 30, serverEntryPointName)
-			serverEntryPoint.httpServer.BlockingClose()
-			cancel()
-		}()
-		<-ctx.Done()
+		log.Debugf("Waiting %d seconds before killing connections on entrypoint %s...", server.globalConfiguration.GraceTimeOut, serverEntryPointName)
```

Comment:
In fact, it was not working in the old version because of `<-ctx.Done()`.
I used a `WaitGroup` instead.